cleantalk.php

```diff
@@ -447,6 +448,42 @@ function apbct_alt_session__save__WP_AJAX()
 );
 new  \Cleantalk\Antispam\Integrations($apbct_active_integrations, (array)$apbct->settings);
 
+$jsErrorsArr = apbct_check_post_for_no_cookie_data();
+if ($jsErrorsArr && $jsErrorsArr['data']) {
+    apbct_write_js_errors($jsErrorsArr['data']);
+}
+
+/**
+ * @psalm-suppress UnusedVariable
+ */
+function apbct_write_js_errors($data)
+{
+    $tmp = substr($data, strlen('_ct_no_cookie_data_'));
+    $errors = json_decode(base64_decode($tmp), true);
+    if (!isset($errors['ct_js_errors'])) {
+        return;
+    }
+    $errors = $errors['ct_js_errors'];
+    $existErrors = get_option(APBCT_JS_ERRORS);
```

Comment:
please no camel case in variables name for PHP

Reply:
fixed